src/Toolkit/F.php

```diff
@@ -105,8 +108,35 @@ class F
         ],
     ];
 
+    /**
+     * @var array
+     */
+    public static $resizable = [
+        'jpg',
+        'jpeg',
+        'gif',
+        'png',
+        'webp'
+    ];
+
+    /**
+     * @var array
+     */
     public static $units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
 
+    /**
+     * @var array
+     */
+    public static $viewable = [
+        'avif',
```

Comment:
`AVIF` added to viewable as default. Should it be in the core or extended? #2877